.npmignore

```diff
@@ -0,0 +1 @@
+.asf.yaml
```

Comment:
I think we should use
```suggestion
.*
```
instead, like in our other repos. That will also remove the `.eslintrc` from the bundle which is only the lint configuration for this repo, _not_ the config we export.